### PR TITLE
Create new rit show env command

### PR DESCRIPTION
**Pull request by kaduartur**
### Description
The `rit show context`  command has been deprecated and we will now use the new `rit show env` command which has the same functionality as the old command.

### How to verify it
It's a small part of the #554 issue, a separate pull request was created to facilitate code review

### CODE REVIEW

File: pkg/cmd/mocks_test.go [hunk 1]
```diff
@@ -211,16 +211,16 @@ func (ctxSetterMock) Set(ctx string) (env.Holder, error) {
...
 
-type ctxFinderMock struct{}
+type envFinderMock struct{}
 
-func (ctxFinderMock) Find() (env.Holder, error) {
+func (envFinderMock) Find() (env.Holder, error) {
 	return env.Holder{}, nil
 }
 
 type ctxFindRemoverMock struct{}
```
Comment (henriquemoraeszup) [inline pkg/cmd/mocks_test.go:220]: shouldnt this be renamed to `env` as well?

File: pkg/cmd/show.go [hunk 2]
```diff
@@ -16,15 +16,31 @@
...
 		Args:      cobra.OnlyValidArgs,
 	}
+
+	deprecatedMsg := fmt.Sprintf(
+		`you can now use the "%v" command for the same purpose as the "%v" command.`,
+		prompt.Bold("rit show env"),
+		prompt.Bold("rit show context"),
+	)
+
+	DeprecateCmd(cmd, "context", deprecatedMsg)
```
Comment (henriquemoraeszup) [inline pkg/cmd/show.go:43]: just curious, even with only `env` and `formula-runner` as valid args, `context` will still show the deprecation correctly?

**Review by henriquemoraeszup — APPROVED** [review]
Just some small comments, but im good to go 🚀

File: pkg/credential/path.go [hunk 3]
```diff
@@ -17,18 +17,15 @@
...
 
 import (
-	"fmt"
+	"path/filepath"
 )
 
-const (
-	pathPattern     = "%s/credentials/%s"
-	credFilePattern = "%s/%s"
-)
```
Comment (victorschumacherzup) [inline pkg/credential/path.go:26]: nice

**Review by victorschumacherzup — APPROVED** [review]

File: pkg/cmd/show_env_test.go [hunk 4]
```diff
@@ -21,9 +21,9 @@ import (
 )
 
 func TestNewShowContextCmd(t *testing.T) {
```
Comment (brunasilvazup) [inline pkg/cmd/show_env_test.go:23]: Typo here too `TestNewShowContextCmd` -> `TestNewShowEnvCmd`?

File: pkg/commands/builder.go [hunk 5]
```diff
@@ -208,7 +208,7 @@ func Build() *cobra.Command {
 
 	deleteCtxCmd := cmd.NewDeleteContextCmd(ctxFindRemover, inputBool, inputList)
 	setCtxCmd := cmd.NewSetEnvCmd(ctxFindSetter, inputText, inputList)
-	showCtxCmd := cmd.NewShowContextCmd(ctxFinder)
+	showCtxCmd := cmd.NewShowEnvCmd(ctxFinder)
```
Comment (brunasilvazup) [inline pkg/commands/builder.go:211]: Will you change the variables in another PR?

File: pkg/credential/setter.go [hunk 6]
```diff
@@ -20,43 +20,51 @@ import (
...
-	if err := fileutil.CreateDirIfNotExists(dir, 0700); err != nil {
+	dir := Dir(s.homePath, envHolder.Current)
+	if err := s.dir.Create(dir); err != nil {
 		return err
 	}
 
-	credFile := File(s.homePath, ctx.Current, cred.Service)
-	if err := fileutil.WriteFilePerm(credFile, cb, 0600); err != nil {
+	credFile := File(s.homePath, envHolder.Current, cred.Service)
+	if err := s.file.Write(credFile, cb); err != nil {
```
Comment (brunasilvazup) [inline pkg/credential/setter.go:67]: :1st_place_medal:

File: pkg/credential/setter_test.go [hunk 7]
```diff
@@ -76,7 +77,7 @@ func (suite *SetterTestSuite) TestSetCredentialToDefalt() {
 			filePathExpectedCreated := File(suite.HomePath, suite.envHolderDefault.Current, suite.DetailCredentialInfo.Service)
 
 			contextFinderMock.On("Find").Return(t.context, nil)
-			setter := NewSetter(suite.HomePath, contextFinderMock)
+			setter := NewSetter(suite.HomePath, contextFinderMock, dirManager, fileManager)
```
Comment (brunasilvazup) [inline pkg/credential/setter_test.go:80]: Will you change the name tests too? `"Context informed"` and `"Context not informed"`

File: pkg/cmd/mocks_test.go (hunk 1 above)
Comment (kaduartur) [reply pkg/cmd/mocks_test.go:220]: this will be modified in a next PR

File: pkg/cmd/show.go (hunk 2 above)
Comment (kaduartur) [reply pkg/cmd/show.go:43]: Yes, It will still show the deprecation.

File: pkg/credential/setter_test.go (hunk 7 above)
Comment (kaduartur) [reply pkg/credential/setter_test.go:80]: this will be modified in the next PR

File: pkg/commands/builder.go (hunk 5 above)
Comment (kaduartur) [reply pkg/commands/builder.go:211]: this will be modified in the next PR

File: pkg/cmd/show_env_test.go (hunk 4 above)
Comment (kaduartur) [reply pkg/cmd/show_env_test.go:23]: this will be modified in a specific PR to rename variables and function names

**Pull request merged**